### PR TITLE
Fixes imports that are no longer required and causes RTD to fail building

### PR DESCRIPTION
**Pull request by brenthuisman**

### CODE REVIEW

File: doc/scripts/make_images.py [hunk 1]
```diff
@@ -12,6 +10,8 @@
 #
 
 def dataframe_line_plot(input_csv, output_svg):
+    import seaborn
```
Comment (bcumming) [inline doc/scripts/make_images.py:13]: How does this fix the problem? It is still importing the problematic dependencies.

Instead, I thought that we had agreed to include the svg file in the documentation, and completely remove this code for generating the image.

**Review by bcumming — CHANGES_REQUESTED** [review]
The dependency is still there and should be removed.

File: doc/scripts/make_images.py (hunk 1 above)
Comment (brenthuisman) [reply doc/scripts/make_images.py:13]: Yes, I left the function there just in case (it's not called). If you prefer, I can remove it entirely.

Comment (bcumming) [reply doc/scripts/make_images.py:13]: Yes, let's remove it entirely.

**Review by bcumming — APPROVED** [review]

**Pull request merged**